services/src/main/java/org/exoplatform/news/notification/provider/PushTemplateProvider.java

```diff
@@ -20,7 +14,8 @@
     @TemplateConfig(pluginId = LikeNewsNotificationPlugin.ID, template = "war:/notification/templates/push/postNewsNotificationPlugin.gtmpl"),
     @TemplateConfig(pluginId = LikeSharedNewsNotificationPlugin.ID, template = "war:/notification/templates/push/postNewsNotificationPlugin.gtmpl"),
     @TemplateConfig(pluginId = CommentNewsNotificationPlugin.ID, template = "war:/notification/templates/push/postNewsNotificationPlugin.gtmpl"),
-    @TemplateConfig(pluginId = CommentSharedNewsNotificationPlugin.ID, template = "war:/notification/templates/push/postNewsNotificationPlugin.gtmpl") })
+    @TemplateConfig(pluginId = CommentSharedNewsNotificationPlugin.ID, template = "war:/notification/templates/push/postNewsNotificationPlugin.gtmpl"),
+    @TemplateConfig(pluginId = MentionInNewsNotificationPlugin.ID, template = "war:/notification/templates/push/postNewsNotificationPlugin.gtmpl") })
```

Comment:
Shouldn't it be MentionInNewsNotificationPlugin.gtmpl instead of postNewsNotificationPlugin.gtmpl

Reply:
As you see all the cases are made in only one notification template.

Reply:
Ok I see :-)